.github/workflows/build.yml

```diff
@@ -239,20 +240,14 @@ jobs:
           LIB_ANONCREDS_PATH: ../../
         run: pnpm test
 
-      - name: Set NPM config
```

Comment:
This was added for a reason, let's hope it will work with the changes (also the always-auth seems to be removed)

Reply:
Ah yeah good catch. If that causes an issue I will add it back to the `.npmrc`.